collection.php

```diff
@@ -0,0 +1,34 @@
+<?php
+//--QUERY--
+$db = new PDO('mysql:host=db; dbname=collection', 'root', 'password');
```

Comment:
Make sure to set the default fetch mode on your database here, otherwise it will be doing that weird behavior where it gets your results as an index and assoc array.

`$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
`

Reply:
I would then suggest putting this into a function that you can call to get a database connection. That way you can reuse it easily on other pages, for example if you wanted to have a separate add item or edit item page in the future.